docker/ansible-runner/Pipfile

```diff
@@ -6,23 +6,65 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-psutil = "*"
-ansible-runner = "*"
-paramiko = "*"
-ansible = "==2.10.7"
-ssh-agent-setup = "*"
-pywinrm = "*"
-requests-ntlm = "*"
-hcloud = "*"
-pyvmomi = "*"
-openshift = "*"
-pyyaml = "*"
-azure-mgmt-compute = "*"
-azure-mgmt-storage = "*"
-azure-mgmt-resource = "*"
-azure-mgmt-network = "*"
-dnspython = "*"
-footmark = "*"
+psutil = "==5.8.0"
+ansible-runner = "==1.4.7"
+paramiko = "==2.7.2"
+ansible = ">=2.10.7,<2.11.0"
+ssh-agent-setup = "==1.0.1"
+pywinrm = "==0.4.2"
+hcloud = "==1.12.0"
+pyvmomi = "==7.0.2"
+openshift = "==0.12.1"
+kubernetes = "==12.0.1"
+dnspython = "==2.1.0"
+footmark = "==1.20.0"
+azure-cli-core = "==2.11.1"
+azure-cli-nspkg = "==3.0.2"
+azure-common = "==1.1.11"
+azure-mgmt-apimanagement = "==0.2.0"
+azure-mgmt-authorization = "==0.51.1"
+azure-mgmt-batch = "==5.0.1"
+azure-mgmt-cdn = "==3.0.0"
+azure-mgmt-compute = "==10.0.0"
+azure-mgmt-containerinstance = "==1.4.0"
+azure-mgmt-containerregistry = "==2.0.0"
+azure-mgmt-containerservice = "==9.1.0"
+azure-mgmt-dns = "==2.1.0"
+azure-mgmt-keyvault = "==1.1.0"
+azure-mgmt-marketplaceordering = "==0.1.0"
+azure-mgmt-monitor = "==0.5.2"
+azure-mgmt-managedservices = "==1.0.0"
+azure-mgmt-managementgroups = "==0.2.0"
+azure-mgmt-network = "==12.0.0"
+azure-mgmt-nspkg = "==2.0.0"
+azure-mgmt-privatedns = "==0.1.0"
+azure-mgmt-redis = "==5.0.0"
+azure-mgmt-resource = "==10.2.0"
+azure-mgmt-rdbms = "==1.9.0"
+azure-mgmt-search = "==3.0.0"
+azure-mgmt-servicebus = "==0.5.3"
+azure-mgmt-sql = "==0.10.0"
+azure-mgmt-storage = "==11.1.0"
+azure-mgmt-trafficmanager = "==0.50.0"
+azure-mgmt-web = "==0.41.0"
+azure-nspkg = "==2.0.0"
+azure-storage = "==0.35.1"
+msrest = "==0.6.10"
+msrestazure = "==0.6.4"
+azure-keyvault = "==1.0.0a1"
+azure-graphrbac = "==0.61.1"
+azure-mgmt-cosmosdb = "==0.5.2"
+azure-mgmt-hdinsight = "==0.1.0"
+azure-mgmt-devtestlabs = "==3.0.0"
+azure-mgmt-loganalytics = "==1.0.0"
+azure-mgmt-automation = "==0.1.1"
+azure-mgmt-iothub = "==0.7.0"
+azure-mgmt-recoveryservices = "==0.4.0"
+azure-mgmt-recoveryservicesbackup = "==0.6.0"
+azure-mgmt-eventhub = "==2.0.0"
+requests-ntlm = "==1.1.0"
+PyYAML = "==5.4.1"
+vsphere-automation-sdk = {file = "https://github.com/vmware/vsphere-automation-sdk-python/archive/v7.0.1.0.tar.gz"}
 
 [requires]
 python_version = "3.7"
```

Comment:
@SergeBakharev  Let's change this to 3.9. You will need to regenerate the lock file after this change.